### PR TITLE
BAEL-5386: Format Instant to String in Java

### PR DESCRIPTION
**Pull request by azhwani**
Simple PR

### CODE REVIEW

File: core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    @Test
+    public void givenInstant_WhenUsingDateTimeFormatter_thenFormat() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(PATTERN_FORMAT)
+            .withZone(ZoneId.systemDefault());
+
+        Instant instant = Instant.parse("2022-02-15T18:35:24.00Z");
+        String formattedInstant = formatter.format(instant);
+
+        assertThat(formattedInstant).isEqualTo("15.02.2022");
+
```
Comment (KevinGilmore) [inline core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java:26]: For this and other methods, remove the blank line before the closing brace

File: core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.joda.time.format.DateTimeFormat;
+import org.junit.Test;
+
+public class FormatInstantUnitTest {
+    
+    private static final String PATTERN_FORMAT = "dd.MM.yyyy";
+
+    @Test
+    public void givenInstant_WhenUsingDateTimeFormatter_thenFormat() {
```
Comment (KevinGilmore) [inline core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java:18]: Lowercase the "w" in "when" for all the test method names

**Review by KevinGilmore — CHANGES_REQUESTED** [review]
Nice! Just a little formatting/naming cleanup.

File: core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java (hunk 2 above)
Comment (azhwani) [reply core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java]: done

File: core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java (hunk 1 above)
Comment (azhwani) [reply core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java]: done

**Review by KevinGilmore — APPROVED** [review]

**Pull request merged**